src/app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def title
+    if content_for?(:title)
```

Comment:
I think it could look like this 
```ruby
Array[content_for(:title)].prepend('Ruby Functions').join(' - ')
```

Reply:
looks like join return `quotes` and rails template engine doesn't handle them as expected
![image](https://user-images.githubusercontent.com/8272048/78180371-fe487580-7462-11ea-8235-764638909732.png)

Reply:
the `Array[....]` is not working as I expected, I wanted this part to convert the value to an array if it's not an array already, to support single item usage and array.

looks like the correct line should be 
`Array()` not `Array[]`
https://stackoverflow.com/questions/18358717/ruby-elegantly-convert-variable-to-an-array-if-not-an-array-already

Reply:
still the same issue. also I would keep the `sprintf` so any view can have it's own format whether it's `foo >> bar` or` foo - bar >> action`